plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaFilterManager.java

```diff
@@ -40,6 +40,7 @@
 import java.util.List;
```

Comment:
@motto1314 Welcome in the community! Thank you for your contribution.

Can you please make sure you read and follow:
 - https://github.com/trinodb/trino/blob/master/CONTRIBUTING.md - we are not allowed to accept any change without CLA signed.
 - https://github.com/trinodb/trino/blob/master/DEVELOPMENT.md
 
 Can you please improve your commit message and explain add a better commit message and pull request title and description. Please mention what is the expected user experience change. 

It looks like you are adding a new functionality, can you please also add proper testing of that feature to `TestKafkaIntegrationSmokeTest`?

Reply:
thank you very much.
1、CLA has been sent to cla@trino.io , please check
2、DEVELOPMENT.md I will study it carefully
3、I'll consider adding a proper testing